### ISSUE TITLE
Cannot save pdf

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#841]
```
What steps will reproduce the problem?
1.from chrome open a PDF from the web
2.ebookdroid popup and open the pdf
3.exiting or pressing back button the PDF close without ask to save

What is the expected output?
Ask to save as before last update


What do you see instead?
The app close.

What version of the product are you using?
2.1.10

On what operating system?
Android 4.4.4

What device are you using?
Nexus7

Please provide any additional information below.

```

Original issue reported on code.google.com by `demartin...@gmail.com` on 19 Aug 2014 at 2:26

**Comment by GoogleCodeExporter** [comment]
```
Issue 842 has been merged into this issue.
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 21 Aug 2014 at 10:25

**Comment by GoogleCodeExporter** [comment]
```
Try an appropriate new version:
https://drive.google.com/folderview?id=0B6DTNNagrvFPZkRwWmJOekoyNVk&usp=sharing
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 21 Aug 2014 at 11:23
- Changed state: **Discussion**

**Comment by GoogleCodeExporter** [comment]
```
Issue 846 has been merged into this issue.
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 25 Aug 2014 at 5:47

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.1.11
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 25 Aug 2014 at 5:48
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Good. I wait for the new official version. When will it be available on Play 
Store ?
```

Original comment by `paolopi...@gmail.com` on 25 Aug 2014 at 1:01
